### PR TITLE
order workflow

### PR DESCRIPTION
**Pull request by Kenneth-Aponte**

### CODE REVIEW

File: frontend/src/shared/providers/CartProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+  PropsWithChildren,
+  createContext,
+  useContext,
+  useMemo,
+} from "react";
+import { Combo, Item } from "../../../../backend/src/menu/menu.dto";
+import { useOrderService } from "../hooks/useOrderService";
+import { useRouter } from "next/router";
+
+//TODO: FIX TYPES
```
Comment (isabellagarrido) [inline frontend/src/shared/providers/CartProvider.tsx]: Is this todo still pending?

File: backend/prisma/schema.prisma [hunk 2]
```diff
@@ -28,13 +28,14 @@ model User {
...
+  userId          Int
   status          String        @default("pending")
+  totalPrice      Float
   createdAt       DateTime      @default(now())
   updatedAt       DateTime      @default(now())
   removed         Boolean       @default(false)
-  orderItems      OrderItem[]
-  orderCombos     OrderCombo[]
+  items           OrderItem[]
+  combos          OrderCombo[]
```
Comment (isabellagarrido) [inline backend/prisma/schema.prisma]: Can we rename these back to orderItems/orderCombos?

File: frontend/src/pages/customers/menu/index.tsx [hunk 3]
```diff
@@ -41,20 +44,30 @@ export default function MenuPage() {
...
+            <Typography variant="h2" mb={2} textAlign={"center"}>
+              {activeMenu.name}
             </Typography>
             <Button
               variant="contained"
               sx={{ maxHeight: 50 }}
               onClick={() => router.push("/customers/order")}
             >
-              Order Now
+              Order Now!
```
Comment (isabellagarrido) [inline frontend/src/pages/customers/menu/index.tsx]: Punctuation in a button is a little weird. What do you think? 

```suggestion
              Order Now
```

File: frontend/src/shared/components/baseDialog.tsx [hunk 4]
```diff
@@ -68,14 +78,18 @@ export default function BaseDialog({
       open={open}
       onClose={handleClose}
       aria-labelledby="responsive-dialog-title"
+      fullWidth
+      PaperProps={{ style: { backgroundColor: "#f8f8fc" } }}
```
Comment (isabellagarrido) [inline frontend/src/shared/components/baseDialog.tsx:82]: How does this background color look on other modals?

File: frontend/src/shared/providers/CartProvider.tsx [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+        };
+      });
+
+      await createOrder({
+        items,
+        combos,
+        totalPrice,
+      });
+      formik.setValues(cartInitialValues);
+      //TODO: Change this to the new order status page
```
Comment (isabellagarrido) [inline frontend/src/shared/providers/CartProvider.tsx:114]: Is this todo still valid?

File: frontend/src/shared/components/baseDialog.tsx (hunk 4 above)
Comment (Kenneth-Aponte) [reply frontend/src/shared/components/baseDialog.tsx:82]: lookd good trust me

File: frontend/src/shared/providers/CartProvider.tsx (hunk 5 above)
Comment (Kenneth-Aponte) [reply frontend/src/shared/providers/CartProvider.tsx:114]: yea, for the next pr that i will work on :)

**Review by isabellagarrido — APPROVED** [review]

**Pull request merged**